packages/cli/src/commands/start/startBackend.ts

```diff
@@ -26,33 +26,56 @@ interface StartBackendOptions {
 }
 
 export async function startBackend(options: StartBackendOptions) {
-  const hasDev = await fs.pathExists(paths.resolveTarget('dev'));
-  if (hasDev) {
+  if (!process.env.LEGACY_BACKEND_START) {
     const waitForExit = await startBackendExperimental({
-      entry: 'dev/index',
+      entry: 'src/index',
       checksEnabled: false, // not supported
       inspectEnabled: options.inspectEnabled,
       inspectBrkEnabled: options.inspectBrkEnabled,
     });
 
     await waitForExit();
-  } else if (!process.env.LEGACY_BACKEND_START) {
-    const waitForExit = await startBackendExperimental({
+  } else {
+    const waitForExit = await cleanDistAndServeBackend({
       entry: 'src/index',
+      checksEnabled: options.checksEnabled,
+      inspectEnabled: options.inspectEnabled,
+      inspectBrkEnabled: options.inspectBrkEnabled,
+    });
+
+    await waitForExit();
+  }
+}
+
+export async function startBackendPlugin(options: StartBackendOptions) {
+  if (!process.env.LEGACY_BACKEND_START) {
+    const hasEntry = await fs.pathExists(paths.resolveTarget('dev'));
```

Comment:
Perhaps we can check if `src/run.ts` exists as well? Then we can provide a more accurate suggestion, since using `LEGACY_BACKEND_START=1` would only make sense if `src/run.ts` exists. Basically if `src/run.ts` but no `dev/index.ts` -> suggest to run with `LEGACY_BACKEND_START=1`, but if neither of them are available, tell the user to create `dev.index.ts`, similar to the message below.

Reply:
I've added more warnings 😄